### PR TITLE
Add responsiveness

### PR DESCRIPTION
**Pull request by Maianki**
### This PR includes:
- Files for sidebar drawer for smaller viewport 
- css and jsx for filter component for smaller viewport
- media queries for different pages to make them responsive for different viewport

## Screenshots of the screens
Screenshot (706)
Screenshot (705)

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Checked responsiveness works fine
code looks clean and understandable
good to merge

File: src/components/Snackbar/snackbar.css [hunk 1]
```diff
@@ -8,8 +8,8 @@
 .snackbar-container {
     font-size: 14px;
     position: fixed;
-    z-index: var(--Z-INDEX-99);
-    top: 7.5rem;
+    z-index: var(--Z-INDEX-999);
```
Comment (sainath7878) [inline src/components/Snackbar/snackbar.css:11]: Appreciation: variables for z-index!

File: src/index.css [hunk 2]
```diff
@@ -43,4 +41,30 @@ code {
 
 .card img{
   aspect-ratio: 4/3;
+}
+/*MEDIA QUERIES*/
+
+@media screen and (max-width:648px) {
```
Comment (sainath7878) [inline src/index.css:45]: since variables are defined for breakpoints, you could use them

**Review by sainath7878 — APPROVED** [review]
Suggested a change. Apart from the suggestion, everything looks good. However, if the user is logged out, there is no option for the user to login back in smaller viewports.

**Comment by Maianki** [comment]
Thanks for reviewing @sainath7878 @srishtigautam21 I have done the said changes. Merging PR.

**Pull request merged**